oss_src/unity/lib/unity_sarray_builder.cpp

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (C) 2015 Dato, Inc.
+ * All rights reserved.
```

Comment:
There appear to be 2 copies of this file. sarray_builder.cpp (above in unity/extensions) and this file?